### ISSUE TITLE
Audit UIDS for accessibility issues after Storybook update

### ISSUE THREAD

**Issue by briand44** [uiowa/uids#1045]
# Motivation

Marie previously did an accessibility review of UIDS and identified some issues needing remediation (https://github.com/uiowa/uiowa/issues/9295#issuecomment-3599058905). Since then we upgraded Storybook and addressed some of these accessibility issues (https://github.com/uiowa/uids/pull/1042).

It would be good to do another accessibility audit/review of UIDS to identify any new or remaining issues that need resolved.

# Additional Info

Should re-trigger site improve scan once Storybook 10 deployed.

Some previous items that have been mentioned:

- Interactive element does not meet minimum size nor spacing (AA) - consistently flagging on https://uids.brand.uiowa.edu/?path=/story/showcase-forms--information-request-form for me. Sometimes happens on other pages like https://uids.brand.uiowa.edu/?path=/docs/introduction--docs but not reliably.
  - Interactive story playground had two links. The introduction has links to the stylesheets that are apparently too small still?
- Text is clipped when resized - this is flagged on Set Strings option when no string has been set - see any headline story like https://uids.brand.uiowa.edu/?path=/story/components-headline--default for this behavior. This is also flagged when tablet view is used, but I believe that's expected given how the view is set up?
- Link missing text alternative - flagged on https://uids.brand.uiowa.edu/?path=/story/components-button--no-text and https://uids.brand.uiowa.edu/?path=/docs/components-button--docs for buttons with no text.
This is a component issue for our no-text button. Out of the context of a card, they don't have an accessible name associated with them
- Controls tab identified as data table with the TH cells lacking the SCOPE attribute
  - I can't reproduce this with live, latest, local.
- Siteimprove is flagging no skip link, but the skip link checker identifies it fine.
- No headings exist, but based on Siteimprove flagging this as a best practice rather than a compliance issue, it's fine for now.
- Noticeable new issue: aria-label="Collapse/Expand" with visible text "Components/Elements/Showcase" fails Label in Name, but there is no way to customize without doing a mutationObserver.

**Comment by briand44** [comment]
## Grooming
 
**Quadrant:** 

Complicated

 **Discussion:**
 - new deadline 1 year out
 - may have more flexibility with upstream issues with an extended deadline
 - Siteimprove scan has been re-triggered post Storybook update
 - Not sure of future of UIDS or timeline
 - Scope to audit and understanding outstanding issues, if Codex/Claude can help resolve low hanging fruit, great
 - larger issues captured so we can have further conversations around investment

**Proposed solution or definition of done:**
- Review/document any reported issues in Siteimprove
- Manually review and document any other accessibility issues
- Feel free to address any low hanging issues but not required for definition of done
